src/test/unit/java/com/emt/service/MatchServiceTest.java

```diff
@@ -20,50 +22,77 @@
+    @Test
+    @SuppressWarnings("PMD.AvoidDuplicateLiterals")
+    void createMatch_WhenWinnerAndLoserAreIdentical_ShouldThrowException() {
+        CreateMatchRequest request = CreateMatchRequest.builder().winnerId(1L).loserId(1L).build();
 
-    assertThatThrownBy(() -> matchService.createMatch(request))
-        .isInstanceOf(IdenticalPlayersException.class)
-        .hasMessageContaining("A match cannot be created with identical players.");
-  }
+        assertThatThrownBy(() -> matchService.createMatch(request))
+                .isInstanceOf(IdenticalPlayersException.class)
+                .hasMessageContaining("A match cannot be created with identical players.");
+    }
 
-  @Test
-  void createMatch_ShouldCreateMatchSuccessfully_WhenPlayersAreDifferent() {
-    CreateMatchRequest request = CreateMatchRequest.builder().winnerId(1L).loserId(2L).build();
+    @Test
+    void createMatch_WhenPlayersAreDifferent_ShouldCreateMatchSuccessfully() {
+        CreateMatchRequest request = CreateMatchRequest.builder().winnerId(1L).loserId(2L).build();
 
-    Player winner = new Player(1L, "WinnerPlayer", 2500, Instant.now());
-    Player loser = new Player(2L, "LoserPlayer", 2000, Instant.now());
-    Match match = new Match();
-    MatchResponse expectedResponse =
-        new MatchResponse("WinnerPlayer", "LoserPlayer", Instant.now());
+        Player winner = new Player(1L, "WinnerPlayer", 2500, Instant.now());
+        Player loser = new Player(2L, "LoserPlayer", 2000, Instant.now());
+        Match match = new Match();
+        MatchResponse expectedResponse =
+                new MatchResponse("WinnerPlayer", "LoserPlayer", Instant.now());
 
-    given(playerService.getReferenceById(1L)).willReturn(winner);
-    given(playerService.getReferenceById(2L)).willReturn(loser);
-    given(matchMapper.mapToEntity(winner, loser)).willReturn(match);
-    given(matchRepository.save(match)).willReturn(match);
-    given(matchMapper.mapToResponse(match)).willReturn(expectedResponse);
+        given(playerService.getPlayerById(1L)).willReturn(winner);
+        given(playerService.getPlayerById(2L)).willReturn(loser);
+        given(matchMapper.mapToEntity(winner, loser)).willReturn(match);
+        given(matchRepository.save(match)).willReturn(match);
+        given(matchMapper.mapToResponse(match)).willReturn(expectedResponse);
 
-    MatchResponse actualResponse = matchService.createMatch(request);
+        MatchResponse actualResponse = matchService.createMatch(request);
+
+        assertThat(actualResponse).isEqualTo(expectedResponse);
+        verify(matchRepository).save(match);
+    }
+
+    @ParameterizedTest(name = "[{index}] Calculate correctly Elo-Ranking between two players with winner rating {0} and loser rating {1}.")
+    @CsvSource({
+            "1200, 1100",
+            "1200, 1250",
+            "1300, 1200",
+            "1500, 1200",
+            "1200, 1000"
+    })
```

Comment:
To make it explicit, you can add a parameterized parameter `ratingChange`, to verify raiting change using both formula and this hardcoded value